src/style.css

```diff
@@ -1,3 +1,99 @@
+html {
+  margin: 0;
+  font-family: roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+ul {
+  padding: 0;
+  list-style: none;
+}
+
+li {
+  list-style: none;
+}
+
+.img {
+  position: absolute;
+  top: 10px;
+  left: 50px;
+}
+
+.header {
+  display: flex;
+
+  background-color: #fff;
+
+  justify-content: end;
+  align-items: center;
+}
+
+.nav {
+  /* background-color: chartreuse; */
+
+  width: 639px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  width: 100%;
+  align-items: center;
+
+  /* background-color: blueviolet; */
+
+  justify-content: space-between;
+}
+
+.nav__iten {
```

Comment:
There is a typo in the class name '.nav__iten'. It should be '.nav__item' to match the HTML class names and ensure the styles are applied correctly.